pkg/cli/admin/verifyimagesignature/verify-signature.go

```diff
@@ -106,7 +109,7 @@ func NewCmdVerifyImageSignature(f kcmdutil.Factory, streams genericiooptions.IOS
 		Example: verifyImageSignatureExample,
 		Run: func(cmd *cobra.Command, args []string) {
 			kcmdutil.CheckErr(o.Validate())
-			kcmdutil.CheckErr(o.Complete(f, cmd, args))
+			ocmdhelpers.CheckOAuthDisabledErr(o.Complete(f, cmd, args), o.DiscoveryClient)
```

Comment:
It is interesting to see this command but after checking, indeed it relies on user object.